Chapter1/index.rst

```diff
@@ -1,8 +1,8 @@
 Chapter 1: Understanding Climate Science and Carbon Models
-==========================================================
+===========================================================
 Carbon models are tools for evaluating how disturbances, whether man-made or natural, affect the forest ecosystems surrounding us. Understanding these disturbances provides us with important information about the future of our landscapes and the steps to take to lessen their negative consequences.
```

Comment:
**Thought as a reader:** "What is a disturbance? Let me google-- Oh it means 'the interruption of a settled and peaceful condition.' What kind of disturbances are they talking about here?"

**Suggestion**: I think it would be good to add what a 'disturbance' means with the help of an example. Because this term comes up in the code as well.